lib/crowdnode.js

```diff
@@ -18,15 +18,18 @@ CrowdNode._dashApi = Dash.create({ insightApi: CrowdNode._insightApi });
 
 CrowdNode.main = {
   baseUrl: "https://app.crowdnode.io",
+  knowledge: "https://knowledge.crowdnode.io",
```

Comment:
Will you open a separate PR for the knowledge base URL addition please? I'll be happy to merge that immediately.

Reply:
Done, see https://github.com/dashhive/crowdnode.js/pull/40